cmake/modules/dts.cmake

```diff
@@ -233,6 +238,18 @@ if(SUPPORTS_DTS)
     message(STATUS "Generated device_extern.h: ${DEVICE_EXTERN_H}")
   endif()
 
+
+  execute_process(
+    COMMAND ${PYTHON_EXECUTABLE} ${GEN_DRIVER_KCONFIG_SCRIPT}
```

Comment:
question: we are forcing to re-run configure stage if we add a new compatible, is it like this now?